pkg/kubectl/get_test.go

```diff
@@ -42,4 +42,8 @@ func TestGetByKind(t *testing.T) {
 	if out != expects {
 		t.Errorf("%s != %s", out, expects)
 	}
+
+	if out, _ := Client.GetByKind("pods", "foo", "helm"); out != expects {
```

Comment:
Looking at the `GetByKind` impl for the mock. I was just adding a test because I changed the order that those were added to the kubectl command.